website/blog/apr-2023/silk2170.md

```diff
@@ -0,0 +1,167 @@
+---
+{
+    "TableOfContents": {
+        "Name": "Announcing Silk.NET 2.17: Cleaner Code, Smoother Experience.",
+        "Url": "silk2170.html",
+        "Metadata": {
+            "AuthorGitHub": "Perksey",
+            "DateTimeWritten": "02/04/2023 12:30",
+            "PreviewImage": "images/blog/apr-2023/bigmarkattemptdark.png"
+        }
+    }
+}
+---
+
+# Announcing Silk.NET 2.17: Cleaner Code, Smoother Experience.
+
+Are you tired of wading through messy code to achieve the smooth performance you need for your GPU applications? If so, it's time to upgrade to Silk.NET 2.17. This latest update delivers faster, cleaner code and a smoother experience than ever before, making it the easiest way to achieve top-tier performance without sacrificing usability. With Silk.NET 2.17, you can experience the power of clean code, achieve silky-smooth performance, and write code that feels right at home with the rest of your C# code.
+
+Reworked DirectX bindings, WebGPU bindings, Direct2D, DirectComposition, DirectWrite, D3D11-on-12, Android 12, Windows on ARM64 support, better AOT support, drop-in DXVK support, and a whole lot of enhancements & bugfixes. There's something for everyone in Silk.NET 2.17, the largest user experience improvement update to date. Find out what's new in 2.17!
```

Comment:
This is quite weirdly worded, i think WebGPU bindings shouldnt be grouped into "reworked", it should be specifically like "Reworked DirectX bindings, Added WebGPU, D2D, DC, DWrite, and D3D11-on-12 bindings, Android 12 support... so on and so forth

Reply:
Given it another go, feel free to update if this is still not to your liking :)